luthier/src/test/groovy/com/yahoo/bard/webservice/config/luthier/ResourceNodeSupplierTest.groovy

```diff
@@ -0,0 +1,52 @@
+package com.yahoo.bard.webservice.config.luthier
+
+import com.fasterxml.jackson.databind.node.ArrayNode
+import com.fasterxml.jackson.databind.node.ObjectNode
+import spock.lang.Specification
+
+class ResourceNodeSupplierTest extends Specification {
+    ResourceNodeSupplier defaultResourceNodeSupplier
+    ResourceNodeSupplier testResourceNodeSupplier
+    void setup() {
+        defaultResourceNodeSupplier = new ResourceNodeSupplier(null)
+        testResourceNodeSupplier = new ResourceNodeSupplier("DimensionConfig.json")
+    }
+
+    void cleanup() {
+
+    }
+
+    def "Node Existence"() {
+        when:
+        ObjectNode defaultNode = defaultResourceNodeSupplier.get()
+        ObjectNode dimensionNode = testResourceNodeSupplier.get()
+        then:
+        defaultNode == null
+        dimensionNode != null
```

Comment:
We typically indent when and then blocks. Makes things more readable, even if it
isn't syntactically required. So something like:
when:
ObjectNode defaultNode = ...
then:
defaultNode == null

Reply:
Done.